### PR TITLE
`document` - fix examples in `compute` documents

### PR DESCRIPTION
**Pull request by myc2h6o**
fixing issue with `terraform validate`

### CODE REVIEW

File: website/docs/r/virtual_machine_scale_set_extension.html.markdown [hunk 1]
```diff
@@ -28,6 +28,13 @@ resource "azurerm_linux_virtual_machine_scale_set" "example" {
   admin_username      = "adminuser"
   instances           = 1
 
+  source_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "16.04-LTS"
```
Comment (jackofallops) [inline website/docs/r/virtual_machine_scale_set_extension.html.markdown:34]: This version is EOL by Canonical, is it worth bumping to `20.04-LTS` ?

**Review by jackofallops — CHANGES_REQUESTED** [review]
Thanks @myc2h6o - This looks mostly good to me, but one question to look into if you could please? 

Thanks!

File: website/docs/r/virtual_machine_scale_set_extension.html.markdown (hunk 1 above)
Comment (myc2h6o) [reply website/docs/r/virtual_machine_scale_set_extension.html.markdown]: sure, I've updated the versions to 20.04

**Review by katbyte — APPROVED** [review]
LGTM 🔧

**Pull request merged**